### DISCUSSION TITLE
Exclude paths from catch all server route

### DISCUSSION THREAD

**Discussion by pmwmedia** [nuxt/nuxt#26889] [Questions] [answered]
How can I exclude paths from a catch all server route?

File: `server/routes/[language]/example.ts`

```typescript
export default defineEventHandler(async (event) => {
  const language = getRouterParam(event, 'language') || ''
  const supported = ['en', 'de', 'fr'] // Loaded dynamically in real life

  if (supported.includes(language)) {
    return 'Language is supported and will be handled ...'
  } else {
    // TODO: Let Nuxt3 handle the request and serve the normal Vue website
  }
})
```

If the `language` parameter contains a supported value, I want to serve a computed text content. This works so far. However, how can I let Nuxt3 handle the request and serve the normal Vue website, if the `language` parameter does not contain a supported value?

**Comment by meirroth** [comment]
@pmwmedia Hey, have you figured this out? I ran into the same issue 😬

**Comment by TheAlexLichter** [comment, marked as answer]
See https://github.com/nuxt/nuxt/issues/27105

**Reply by meirroth** [reply]
Nice. Thank you 🙏
